xiilib/_gunicorn/webserver.py

```diff
@@ -240,5 +180,5 @@ def _prepare_log_dir(self) -> None:
             self._charm_state.application_error_log_file,
         ):
             log_dir = str(log.parent.absolute())
-            if not container.isdir(log_dir):
-                container.make_dir(log_dir, make_parents=True)
+            if not container.exists(log_dir):
+                container.make_dir(log_dir, make_parents=True, user="_daemon_", group="_daemon_")
```

Comment:
I'm wondering if the `_daemon_` user and group can be defined as constant

Reply:
Indeed, I will update this in the next pull request, thanks!